poker-hand-evaluator/src/Main.java

```diff
@@ -0,0 +1,187 @@
+                    break;
+                }
+                scanner.nextLine();
+                for (int i = 0; i < counterOfPlayer; i++) {
+                    String playerName = scanner.nextLine();
+                    Player player = new Player(playerName);
+                    playerList.add(player);
+                }
+
+                for (Player player : playerList) {
+                    Dealer.startOfDealing(player, deck);
+                }
+
+                Dealer.flopOfDealing(deck);
+                Dealer.turnOfDealing(deck);
+                Dealer.riverOfDealing(deck);
+
+                for (Player player : playerList) {
+                    player.setPlayerCardList(CardsOnTheTable.getDesk());
+                    System.out.printf("\nPlayer %s seven card: %s\n", player.getName(), player.getPlayerCardList());
+                }
+                //DML
+                int roundId = setDatabaseRoundId(connection);
+
+                Map<Player, Hand> roundOfHands = new HashMap<>();
+                for (Player player : playerList) {
+                    Hand playerHand = getBestHand(player.getPlayerCardList());
+                    System.out.printf("\nPlayer %s best hand: %s\n", player.getName(), playerHand);
+                    roundOfHands.put(player, playerHand);
+                    System.out.printf("handvalue: %s\n", roundOfHands.get(player).getHandValue());
+                    //DML
+                    insertToPlayersTable(connection, player, playerHand, roundId);
+                }
+
+                winnerHand = getWinnerHand(roundOfHands);
+
+                //DML
+                updateDatabaseRoundTable(connection, roundId);
+
+                System.out.println("\nPlayers: \n" + playerList);
+                System.out.println("How many person will exit the card game?");
+                int exit = scanner.nextInt();
+                if (exit == playerList.size()) {
+                    gameOver = true;
+                    break;
+                }
+                scanner.nextLine();
+                for (int i = 0; i < exit; i++) {
+                    System.out.println("Who exits the card game?");
+                    String name = scanner.nextLine();
+                    playerList.removeIf(player -> player.getName().equals(name));
+                }
+                for (Player player : playerList) {
+                    player.delPlayerCardList();
+                }
+                CardsOnTheTable.clearDesk();
+            }
+            scanner.close();
+        }
+        System.out.println("Game over!");
+    }
+
+    private static void updateDatabaseRoundTable(Connection connection, int roundId) throws SQLException {
+        String handValue = winnerHand.getHandValue().toString();
+        PreparedStatement pds = connection.prepareStatement(UPDATE_ROUND_ID);
+        pds.setString(1, handValue);
+        pds.setInt(2, roundId);
+        System.out.println(pds);
+        pds.executeUpdate();
```

Comment:
You should at least provide some logs if no DB row was updated - or is this never gonna happen?